Frontend/ui-driver/src/Screens/HomeScreen/View.purs

```diff
@@ -1928,7 +1928,7 @@ addAadhaarNumber push state visibility' =
   , stroke $ "1," <> Color.black600
   , padding $ Padding 20 16 20 16
   , gravity CENTER_VERTICAL
-  , onClick push $ const ClickAddAlternateButton
+  , onClick push $ const LinkAadhaarAC
```

Comment:
We can remove `ClickAddAlternateButton` from Controller if it's not used anywhere else.

Reply:
Removed